DepotDownloader/Util.cs

```diff
@@ -115,12 +115,9 @@ public static byte[] AdlerHash(byte[] input)
 
         public static byte[] SHAHash(byte[] input)
```

Comment:
Might as well remove this method and call SHA1.HashData directly where it's used